src/patternfly/components/Chips/chip-group.scss

```diff
@@ -0,0 +1,60 @@
+@import "../../patternfly-utilities";
+
+.pf-c-chip-group {
+  --pf-c-chip-group--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-chip-group--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-chip-group--PaddingLeft: var(--pf-global--spacer--sm);
```

Comment:
Could use a magic padding top/bottom val here, plus margin top/bottom for all `pf-c-chip`s. It would give you the appropriate row/col gap when the chips wrap.

```
--pf-c-chip-group--PaddingTop: calc(#{var(--pf-global--spacer--xs)} / 2);
--pf-c-chip-group--PaddingRight: var(--pf-global--spacer--sm);
--pf-c-chip-group--PaddingBottom: calc(#{var(--pf-global--spacer--xs)} / 2);
--pf-c-chip-group--PaddingLeft: var(--pf-global--spacer--sm);
```
<img width="318" alt="screen shot 2018-11-13 at 9 57 45 pm" src="https://user-images.githubusercontent.com/5385435/48457031-30363a80-e78f-11e8-8f82-9f46b27436e8.png">